x-pack/plugins/cases/common/api/connectors/connector.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+
+import type { ActionType } from '@kbn/actions-plugin/common';
+/**
+ * ActionResult type from the common folder is outdated.
+ * The type from server is not exported properly so we
+ * disable the linting for the moment
+ */
+
+import type { ActionResult } from '@kbn/actions-plugin/server/types';
+import { JiraFieldsRT } from './jira';
+import { ResilientFieldsRT } from './resilient';
+import { ServiceNowITSMFieldsRT } from './servicenow_itsm';
+import { ServiceNowSIRFieldsRT } from './servicenow_sir';
+import { SwimlaneFieldsRT } from './swimlane';
+
+export type ActionConnector = ActionResult;
+export type ActionTypeConnector = ActionType;
+
+export const ConnectorFieldsRt = rt.union([
+  JiraFieldsRT,
+  ResilientFieldsRT,
+  ServiceNowITSMFieldsRT,
+  ServiceNowSIRFieldsRT,
+  rt.null,
+]);
```

Comment:
Is `SwimlaneFieldsRT` missing here?

Reply:
You are right. This type is not used anymore. I will delete it.